applications/Chat/examples/train_prompts.py

```diff
@@ -163,7 +162,8 @@ def main(args):
                                      batch_size=args.ptx_batch_size,
                                      collate_fn=data_collator)
 
-    (actor, actor_optim), (critic, critic_optim) = strategy.prepare((actor, actor_optim), (critic, critic_optim))
+    (actor, actor_optim), (critic, critic_optim), reward_model, initial_model = \
```

Comment:
For small models like `opt-1.3b`, reward model and initial model are not required to be parallized.

Reply:
You can add a comment here.

Reply:
This is fixed in Commit af2ff25.